tests/test_auth/TestControllers.py

```diff
@@ -117,3 +118,59 @@ def test_github_token_validity(self):
         """
         res = github_token_validity('token')
         self.assertEqual(res, False)
+
+class ManageAccount(BaseTestCase):
+
+    def test_edit_username(self):
+        """
+        Tests if username is edited
+        """
+        self.create_user_with_role(
+            self.user.name, self.user.email, self.user.password, Role.admin)
+        with self.app.test_client() as c:
+            response = c.post(
+                '/account/login', data=self.create_login_form_data(self.user.email, self.user.password))
+            response = c.post(
+                '/account/manage',data=dict(
+                    current_password=self.user.password,
+                    name="T1duS",
+                    email=self.user.email
+                ))
+            user = User.query.filter(User.name == "T1duS").first()
+            self.assertNotEqual(user, None)
+
+    def test_edit_email(self):
+        """
+        Tests if user email is edited
+        """
+        self.create_user_with_role(
+            self.user.name, self.user.email, self.user.password, Role.admin)
+        with self.app.test_client() as c:
+            response = c.post(
+                '/account/login', data=self.create_login_form_data(self.user.email, self.user.password))
+            response = c.post(
+                '/account/manage',data=dict(
+                    current_password=self.user.password,
+                    name=self.user.name,
+                    email="valid@gmail.com"
+                ))
+            user = User.query.filter(User.email == "valid@gmail.com").first()
+            self.assertNotEqual(user, None)
```

Comment:
Same, if email is edited, why is it none?

Reply:
assert**Not**Equal